packages/@aws-cdk/pipelines/lib/legacy/pipeline.ts

```diff
@@ -87,6 +87,20 @@ export interface CdkPipelineProps {
   readonly crossAccountKeys?: boolean;
   // @deprecated(v2): switch to default false
 
+
+  /**
+   * Enables KMS key rotation for cross-account keys
+   *
+   * Can only be set if `crossAccountKeys` is not set.
```

Comment:
This comment is not correct, right?

It should be something like:

```suggestion
   * Cannot be set if `crossAccountKeys` was set to `false`.
```